### PR TITLE
Recruitment Process Checklist

### PR DESCRIPTION
**Pull request by orkundenizli**

### CODE REVIEW

File: checklists/recruitment-process.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+category: Human Resources
+date: "2020-03-11"
+tags: ['recruitment', 'search', 'job-posting', 'interview', 'offer']
+description: Follow the steps to find the right candidate. It helps to understand the candidate's technical and profile features  
+---
+
+- [ ] Determine the correct job position  
+Define the targeted position based on the ongoing projects and growth of the team.
+
+- [ ] Search a candidate pool and create job posting
```
Comment (alpcanaydin) [inline checklists/recruitment-process.md]:
```suggestion
- [ ] Build a candidate pool and create job posting  
```

File: checklists/recruitment-process.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+date: "2020-03-11"
+tags: ['recruitment', 'search', 'job-posting', 'interview', 'offer']
+description: Follow the steps to find the right candidate. It helps to understand the candidate's technical and profile features  
+---
+
+- [ ] Determine the correct job position  
+Define the targeted position based on the ongoing projects and growth of the team.
+
+- [ ] Build a candidate pool and create job posting  
+Create a candidate pool by using dedicated tools and platform like Linkedin. Create the most suitable job post in order to reach to best match.
```
Comment (akemalakcay) [inline checklists/recruitment-process.md]:
```suggestion
Create a candidate pool by using dedicated tools and platform like LinkedIn. Create the most suitable job post in order to reach to best match.
```

**Comment by emr** [comment]
Please don't merge this pr before #87 is merged.

File: checklists/recruitment-process.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Recruitment Process
+category: Human Resources
+date: "2020-03-11"
+tags: ['recruitment', 'search', 'job-posting', 'interview', 'offer']
```
Comment (alpcanaydin) [inline checklists/recruitment-process.md]:
```suggestion
tags: ['recruitment',  'job-posting', 'interview', 'offer']
```

File: checklists/recruitment-process.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+description: Follow the steps to find the right candidate. It helps to understand the candidate's technical and profile features
```
Comment (alpcanaydin) [inline checklists/recruitment-process.md]:
```suggestion
description: Follow these steps to find the right candidate. It helps to understand the candidate's technical capabilities and their profile  
```

File: checklists/recruitment-process.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+Inform the candidate about related position's needs and learn more about their past experiences.
+
+- [ ] Forward test case to the candidate  
+Evaluate to candidates who respond to the introduction mail. Ask to complete a small test run in order to observe candidates technical competence. 
+
+- [ ] Share project case work with candidate         
+Ask for more extensive project from the candidate who completes test process successfully. Get the information about the level of expertise.
+
+- [ ] Organize a job interview with the candidate   
+Organize face-to-face interview with Hiring Managers in order share more detailed information about the candidate.
```
Comment (alpcanaydin) [inline checklists/recruitment-process.md]:
```suggestion
Organize a face-to-face interview with Hiring Managers in order to get more detailed information about the candidate.
```

File: checklists/recruitment-process.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+- [ ] Final interview    
+As the last step of the recruitment process, the final interview is held with the participation of the related managers. Make an offer to the suitable candidate.
```
Comment (alpcanaydin) [inline checklists/recruitment-process.md]:
```suggestion
As the last step of the recruitment process, have a final interview with the candidate and the participation related managers.
```

File: checklists/recruitment-process.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+Evaluate to candidates who respond to the introduction mail. Ask to complete a small test run in order to observe candidates technical competence. 
+
+- [ ] Share project case work with candidate         
+Ask for more extensive project from the candidate who completes test process successfully. Get the information about the level of expertise.
+
+- [ ] Organize a job interview with the candidate   
+Organize face-to-face interview with Hiring Managers in order share more detailed information about the candidate.
+
+- [ ] Final interview    
+As the last step of the recruitment process, we set the the final interview with the participation managers and make an offer to the suitable candidate.
```
Comment (alpcanaydin) [inline checklists/recruitment-process.md]:
```suggestion
As the last step of the recruitment process, have a final interview with the candidate and the participation related managers. Make a job offer if the candidate has been found suitable.
```

File: checklists/recruitment-process.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
+---
+title: Recruitment Process
+category: Human Resources
+date: "2020-03-11"
+tags: ['recruitment',  'job-posting', 'interview', 'offer']
+description: Follow the steps to find the right candidate. It helps us understand the candidate's technical capabilities and profile features
```
Comment (alpcanaydin) [inline checklists/recruitment-process.md]:
```suggestion
description: Follow these steps to find the right candidate. It helps to understand the candidate's technical capabilities and their profile  
```

**Pull request merged**